ChainReactionBindings/TestBase/Pages/WishlistPage.cs

```diff
@@ -69,5 +79,22 @@ public IWebElement GetCountItems(HtmlElement el)
 			var element = el.FindElement(By.CssSelector(CountItemsId));
 			return element;
 		}
+
+		public HtmlElement FindItem(string name)
+		{
+			var element = new HtmlElement(By.XPath(string.Format(FindItemId, name)));

```

Comment:
No need to create variable, you can just do
return new HtmlElement(By.XPath(string.Format(FindItemId, name)));

Reply:
done